lib/kiwi_config.rb

```diff
@@ -294,46 +294,48 @@ def apply_services
+        @system_description["services"].each do |service|
+          @sh <<
             if service.state == "on"
-              @sh << "chkconfig #{service.name} on\n"
+              "chkconfig #{service.name} on\n"
             else
-              @sh << "chkconfig #{service.name} off\n"
-            end
-          end
-
-        when "systemd"
-          # possible systemd service states:
-          # http://www.freedesktop.org/software/systemd/man/systemctl.html#Unit%20File%20Commands
-          @system_description["services"].each do |service|
-            case service.state
-              when "enabled"
-                @sh << "systemctl enable #{service.name}\n"
-              when "disabled"
-                @sh << "systemctl disable #{service.name}\n"
-              when "masked"
-                @sh << "systemctl mask #{service.name}\n"
-              when "static"
-                # Don't do anything because the unit is not meant to be
-                # enabled/disabled manually.
-              when "linked"
-                # Don't do anything because linking doesn't mean enabling
-                # nor disabling.
-              when "enabled-runtime"
-              when "linked-runtime"
-              when "masked-runtime"
-                # Don't do anything because these states are not supposed
-                # to be permanent.
-              else
-                raise Machinery::Errors::ExportFailed.new(
-                  "The systemd unit state #{service.state} is unknown."
-                )
+              "chkconfig #{service.name} off\n"
             end
+        end
+      when "systemd"
+        # possible systemd service states:
+        # http://www.freedesktop.org/software/systemd/man/systemctl.html#Unit%20File%20Commands
+        @system_description["services"].each do |service|
+          case service.state
+          when "enabled"
+            @sh << "systemctl enable #{service.name}\n"
+          when "disabled"
+            @sh << "systemctl disable #{service.name}\n"
+          when "masked"
+            @sh << "systemctl mask #{service.name}\n"
+          when "static"
+            # Don't do anything because the unit is not meant to be
+            # enabled/disabled manually.
+          when "linked"
+            # Don't do anything because linking doesn't mean enabling
+            # nor disabling.
+          when "enabled-runtime"
+          when "linked-runtime"
+          when "masked-runtime"
+            # Don't do anything because these states are not supposed
+            # to be permanent.
+          else
+            raise Machinery::Errors::ExportFailed.new(
+              "The systemd unit state #{service.state} is unknown."
+            )
           end
-
-        else
-          raise "Unsupported init system: #{init_system.inspect}."
+        end
+      else
+        Machinery::Ui.warn(
+          "Warning: Containers do not have an init system, so the package default service" \
```

Comment:
If you install a package the service is either enabled or disabled depending on the package configuration.